manifests/install.pp

```diff
@@ -155,13 +195,81 @@
   }
   # If user did not specify $gitlab_download_link
   else {
+    info('No download link specified')
+  }
+  #Check if v7.10.0 or greater (major packaging changes)
+  if $::gitlab::gitlab_branch >= '7.10.0' {
+    case $::gitlab::gitlab_release {
+      undef, 'basic' : {
+        #Install via packagecloud repository
+        #wget required for installed gpg key in packagecloud repo (may want to use stdlib ensure_packages instead
+        package { 'wget':
+          ensure => 'installed',
+        }
+        include packagecloud
+        #Check OS
+        case $::osfamily {
+          'Debian': {
+            packagecloud::repo { "gitlab/gitlab-ce":
+              type    => 'deb',
+              server_address => 'https://packages.gitlab.com',
+              require => Package['wget'],
```

Comment:
I had it there for testing as I believed it to be a requirement to install the keys.  I've tested and verified it's not required.  I pushed up the install.pp without the wget requirement this morning after verifying.